packages/web3-eth-abi/tests/AbiCoderTest.js

```diff
@@ -26,51 +26,51 @@ describe('AbiCoderTest', () => {
     });
 
     it('calls encodeFunctionSignature with a string as parameter', () => {
-        Utils.sha3 = jest.fn(() => {
+        Utils.keccak256 = jest.fn(() => {
             return '0x000000000';
         });
 
         expect(abiCoder.encodeFunctionSignature('functionName')).toEqual('0x00000000');
 
-        expect(Utils.sha3).toHaveBeenCalledWith('functionName');
```

Comment:
Sha3 is staying you should be testing both so it can flag any issues, say the alias got removed etc.

Reply:
@joshstevens19 I think in AbiCoderTest we shouldn't check both conditions. Here, keccack256 is okay.

Reply:
As long as you have cover for sha3 fine! You want to make sure if someone changes that logic it fails and we are alerted. 100% coverage or as close as is a must In a JS lib. In this case it’s probably enough

Reply:
Ya right. I could probably test for both possibilities.

Reply:
Leave that up to you, rest looks fine I will approve it 👍